packages/api/src/controllers/stripe.ts

```diff
@@ -543,17 +565,21 @@ app.post("/subscribe-hackers-to-pay-as-you-go", async (req, res) => {
       ],
     });
     await db.user.update(user.id, {
-      isActiveSubscription: false,
+      migrated: true,
```

Comment:
Glad we're merging all these different flags in one (and having the default being `false` is always safer for any code that forgets to check if it exists!)